Formula/skesa.rb

```diff
@@ -14,18 +12,18 @@ class Skesa < Formula
   end
 
   depends_on :linux
-
-  resource "readme" do
-    url "https://ftp.ncbi.nlm.nih.gov/pub/agarwala/skesa/README.skesa"
-    sha256 "a576a9739799692524c635dab9fda39ddec391f4aec51877224b36f27f984c22"
-  end
+  depends_on "patchelf" => :build
+  depends_on "zlib"
```

Comment:
Add `unless OS.mac?`

Reply:
I assumed `depends_on :linux` would mean I didn't need to do that. ah well!